<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HuffmanZip</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>261</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>261</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="start">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="btnCmpress">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>180</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/btn3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>299</number>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUncompress">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/btn2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="compress">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="btnComChose">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>50</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(75, 75, 75);</string>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leComChose">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>50</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leComAim">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>130</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnComAim">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>130</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>压缩到</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnComsure">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>200</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnComCancel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="uncompress">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="btnUnChose">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>30</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnname">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>90</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>命名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnAim">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>150</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>解压到</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUnChose">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>30</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUnname">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>90</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUnAim">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>150</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnSure">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>210</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnCancel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
